spec/features/passenger_management_spec.rb

```diff
@@ -2,13 +2,58 @@
 
 require 'rails_helper'
 
-feature 'Passenger Management' do
-  scenario 'creating a new passenger' do
+feature 'Passenger Management' do 
+  feature 'as and admin' do 
+    before :each do
+      sign_in(create :user, :admin)
+    end
+    scenario 'creating a new passenger' do 
```

Comment:
Will this `feature` and `scenario` DSL translate well to system tests? It should be fine to just stick to `describe` and `it`, particularly since that gives verbal descriptions of the expected behavior.

Will defer to @Anbranin here.